api/specs.proto

```diff
@@ -321,6 +321,9 @@ message NetworkSpec {
 	// enabled(default case) no manual attachment to this network
 	// can happen.
 	bool attachable = 6;
+	
+	// Ingress indicates this network will provide the routing-mesh.
```

Comment:
Let's mention in the comment that legacy ingress networks won't have this flag set, and instead have the `com.docker.swarm.internal` label, and the name `ingress`.